akka-stream/src/main/scala/akka/stream/javadsl/Graph.scala

```diff
@@ -164,6 +164,16 @@ object Partition {
    *
    * @param outputCount number of output ports
    * @param partitioner function deciding which output each element will be targeted
+   * @param eagerCancel if true, partition cancels upstream if any of its downstreams cancel.
+   */
+  def create[T](outputCount: Int, partitioner: function.Function[T, Int], eagerCancel: Boolean): Graph[UniformFanOutShape[T, T], NotUsed] =
```

Comment:
Simple paramaters before lambda would probably be nicer from the users point of view